### PR TITLE
Removing the pricing from the employers page

### PR DESCRIPTION
**Pull request by moekify**
Removing the pricing from the employers page and references to commission fees from the FAQ, as well as the direct link to pricing from the footer

This is the first step in transitioning towards subscription pricing

### CODE REVIEW

**Review by izabellima — APPROVED** [review]

File: src/views/tech-hiring.js [hunk 1]
```diff
@@ -61,8 +61,6 @@ export default o => {
         rightImage
       />
 
-      <PaymentOptions />
```
Comment (izabellima) [inline src/views/tech-hiring.js:64]: We can also remove the import on line 17 and delete the component and the styles as well.
`src/components/tech-hiring/payment-options.js`
`src/styles/components/payment-option.scss`

Comment (moekify) [reply src/views/tech-hiring.js:64]: I removed the import, but would keep the component as we are planning to bring back a pricing section in some time when we finalized on the pricing options

**Comment by moekify** [comment]
QA
image

File: src/views/tech-hiring.js (hunk 1 above)
Comment (moekify) [reply src/views/tech-hiring.js:64]: After conversation with @narikazu decided to remove all traces and revert the commit if we want to get them back :)

**Review by izabellima — APPROVED** [review]
LGTM

**Pull request merged**